### PR TITLE
fix(FeedVersionViewer.VersionList): Fix rendering of feed versions.

### LINKED ISSUE #791: List of feed versions broken
## Observed behavior (please include a screenshot if possible)

The list of feed versions does not render anymore when clicking on the view-versions-as-list button when viewing a feed version.

## Expected behavior

The list of feed versions should render.

## Steps to reproduce the problem

1. Open a feed version to view
2. Click the "View as list" toggle image
3. Observe that a list of feed versions (each with a download-export-load-delete toolbar) is not rendered.

## Any special notes on configuration used

NA

## Version of datatools-ui and datatools-server if applicable (exact commit hash or branch name)

Datatools-ui after #776, datatools-server after https://github.com/ibi-group/datatools-server/pull/457

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [na] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Companion PR for https://github.com/ibi-group/datatools-server/pull/464.
Fix #791 by expanding the use of `FeedVersionSummary` to `FeedVersionViewer.VersionList`.

### CODE REVIEW

File: lib/manager/components/version/VersionDateLabel.js [hunk 1]
```diff
@@ -4,20 +4,20 @@ import React, {Component} from 'react'
...
     const {version} = this.props
-    const {validationSummary: summary} = version
+    const {validationResult: summary} = version
     if (!summary) return null
     const now = +moment().startOf('day')
-    const start = +moment(summary.startDate)
-    const end = +moment(summary.endDate)
+    const start = +moment(summary.firstCalendarDate)
+    const end = +moment(summary.lastCalendarDate)
     const future = start > now
```
Comment (miles-grant-ibigroup) [inline lib/manager/components/version/VersionDateLabel.js:21]: Did we upgrade moment?

**Review by miles-grant-ibigroup — APPROVED** [review]
All changed functionality works for me!

File: lib/manager/components/version/VersionDateLabel.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/manager/components/version/VersionDateLabel.js:21]: Could that be handled in #750?

Comment (miles-grant-ibigroup) [reply lib/manager/components/version/VersionDateLabel.js:21]: Sadly that just got merged. I was more curious why these fields changed, but it's not that important!

Comment (binh-dam-ibigroup) [reply lib/manager/components/version/VersionDateLabel.js]: Actually with https://github.com/ibi-group/datatools-server/commit/a6a72a7890847b13b4d7b604b09b04427e62c974 I reverted changes to this component.

**Review by daniel-heppner-ibigroup — APPROVED** [review]
All looks good to me!

**Pull request merged**